docs/index.md

```diff
@@ -11,7 +11,7 @@ lakeFS is an open source platform that delivers resilience and manageability to
 
 With lakeFS you can build repeatable, atomic and versioned data lake operations - from complex ETL jobs to data science and analytics.
 
-lakeFS is [API compatible with AWS S3](reference/s3.md) and works seamlessly with all modern data frameworks such as Spark, Hive, AWS Athena, Presto, etc.
+lakeFS is [API compatible with AWS S3](reference/s3.md), Google Cloud Storage (GCS), and works seamlessly with all modern data frameworks such as Spark, Hive, AWS Athena, Presto, etc.
```

Comment:
currently we do not support GCS as an interface/API - we do support it as a storage type to keep the data.